cfgrammar/src/lib/yacc/ast.rs

```diff
@@ -254,8 +259,8 @@ impl GrammarAST {
         Ok(())
     }
 
-    /// Returns tokens, and rules unreachable from the start rule
-    /// by index into the `ast.tokens` and `ast.rules` fields.
+    /// Return the indices of unexpectedely unused rules (relative to ast.rules)
```

Comment:
One 'e' too many in 'unexpectedly'?

Reply:
Fixed in [26d1b6f](https://github.com/softdevteam/grmtools/pull/348/commits/26d1b6f026e96488b5c909739cc8b5f97fb60037)